ros2topic/ros2topic/verb/echo.py

```diff
@@ -322,3 +333,10 @@ def _message_lost_event_callback(message_lost_status):
         f'\n\ttotal count: {message_lost_status.total_count}',
         end='---\n'
     )
+
+
+def clear_terminal():
+    if platform.system() == 'Windows':
+        os.system('cls')
+    else:
+        os.system('clear')
```

Comment:
As far as I can tell from some local testing, we should be able to replace this with an ANSI escape sequence (tested to work on bash, tcsh, zsh, fish, Windows command prompt, and Windows Powershell):

```suggestion
def clear_terminal():
    print('\x1b[H\x1b[2J')
```

Can others try it locally and see if it works for them?